### PR TITLE
Pull in upstream changes from cisagov/skeleton-ansible-role

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description

This pull request pulls in the upstream changes from [cisagov/skeleton-ansible-role](https://github.com/cisagov/skeleton-ansible-role) and makes the build pass.

## 💭 Motivation and Context

@hillaryj created #8, but the existing codebase was so old that it was still using TravisCI.  It makes sense to bring this repository into the modern era before making any real changes.

## 🧪 Testing

All pre-commit hooks and molecule tests pass.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (dav3r) [inline .github/CODEOWNERS:7]: Do we want to trim this list of code owners down a bit?

**Review by dav3r — APPROVED** [review]
👍 with one question.

**Review by hillaryj — APPROVED** [review]
🦴

File: .github/CODEOWNERS
Comment (jsf9k) [reply .github/CODEOWNERS:7]: I think we are all familiar with this one...at least we all will be after @hillaryj's PR is merged.

**Comment by jsf9k** [comment]
> I'm basing this on the fact that the previous test user code in this repo didn't need that GeoIP license key: https://github.com/cisagov/ansible-role-cyhy-commander/blob/develop/terraform/main.tf#L9

You're right.  It does not seem necessary, so I removed it.

**Pull request merged**
